src/v/cluster/topics_frontend.cc

```diff
@@ -108,14 +108,24 @@ std::vector<std::string_view> get_enterprise_features(
         features.emplace_back("tiered storage");
     }
 
-    constexpr auto schema_id_validation_enabled =
+    static constexpr auto key_schema_id_validation_enabled =
```

Comment:
nit:
`static` on the lambdas is probably a pessimisation that can (will?) get in the way of inlining.

Reply:
pessimization of what?

Reply:
lambdas are normally very inlining-friendly.
Here, we are forcing it to create an object with static lifetime which will be used for all following calls.
[Short discussion](https://www.reddit.com/r/cpp_questions/comments/wxa2oa/should_named_lambdas_be_made_static/) with godbolt examples on r/cpp_questions regarding this

Reply:
> Here, we are forcing it to create an object with static lifetime which will be used for all following calls.

I don't think it applies to stateless named lambdas, though.